wp-seo-main.php

```diff
@@ -443,7 +443,15 @@ function wpseo_cli_init() {
 	add_action( 'admin_init', 'yoast_wpseo_missing_filter', 1 );
 }
 
-if ( ! wp_installing() && ( $spl_autoload_exists && $filter_exists ) ) {
+$is_login_page = false;
+if ( isset( $_SERVER['SCRIPT_NAME'] ) ) {
+	$script_name = sanitize_text_field( wp_unslash( $_SERVER['SCRIPT_NAME'] ) );
+	if ( stripos( wp_login_url(), $script_name ) !== false ) {
```

Comment:
@enricobattocchi maybe we should also check that `$script_name` is a non-empty string?

Because with PHP 8+, if the needle in the `stripos` function is an empty string, it will return `int(0)`.

Which means that in a misconfigured server that returns an empty SCRIPT_NAME and is on PHP 8+, we won't be loading Yoast in any page.

Reply:
Nice catch!
What about:
```suggestion
	if ( strlen( $script_name) > 0 && stripos( wp_login_url(), $script_name ) !== false ) {
```

Reply:
Yeap, that does it :) I'm committing it then and continuing with my review